webpack/assets/javascripts/spice.js

```diff
@@ -0,0 +1,75 @@
+import { SpiceMainConn } from '@spice-project/spice-html5';
+import { sprintf, translate as __ } from './react_app/common/I18n';
+
+export { sendCtrlAltDel } from '@spice-project/spice-html5';
+
+let sc = null;
+
+export function startSpice() {
+  const scheme = 'ws://';
```

Comment:
So no support for `wss://`?

Reply:
Looks like I forgot to hit save on my update. I now see the old code was `ws://` only as well. Considering browsers disallow mixed content by default I wonder how many people actually use this.

Reply:
I'm not changing anything we didn't have before :)

Reply:
> Looks like I forgot to hit save on my update. I now see the old code was `ws://` only as well. Considering browsers disallow mixed content by default I wonder how many people actually use this.

TBH I tend to agree, from very quick testing, it doesn't seems to work on libvirt at all? and could only work with oVirt. I'm fine to suggest removing it from the code base as well....

Reply:
@ekohl for the record - https://projects.theforeman.org/issues/22801

Reply:
And completing it: https://github.com/theforeman/foreman/pull/8515. It really was adding 2 lines of code.